project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -46,6 +46,18 @@ object Docker {
     }
   }
 
+  lazy val clientDockerFile: Initialize[Task[Dockerfile]] = Def.task {
+    val artifact = assembly.value
+    val artifactTargetPath = s"/app/${name.value}.jar"
+    new Dockerfile {
+      from("google/cloud-sdk:alpine")
+      run("apk", "--update", "add", "openjdk8-jre")
+      run("gcloud", "components", "install", "--quiet", "app-engine-java", "kubectl")
+      label("CLIO_VERSION", version.value)
+      add(artifact, artifactTargetPath)
+    }
+  }
```

Comment:
Why kubectl?

Reply:
Needed to get authentication from docker while running on GCE.

Reply:
Can you elaborate? (And add a comment explaining why we bundle these things)